src/audio/kpb.c

```diff
@@ -1648,11 +1707,15 @@ static void kpb_buffer_samples(const struct audio_stream __sparse_cache *source,
 		samples_count = KPB_BYTES_TO_S16_SAMPLES(size);
 		samples_offset = KPB_BYTES_TO_S16_SAMPLES(offset);
 		audio_stream_copy_to_linear(source, samples_offset,
-					    sink, 0, samples_count);
+				    sink, 0, samples_count);
```

Comment:
looks like indentation is broken here.

Reply:
still broken - sometimes easier to use ```git gui``` here to discard these lines as white space changes are difficult to see.